xcube_geodb_openeo/core/geodb_datastore.py

```diff
@@ -44,8 +44,12 @@ def geodb(self):
 
         server_url = self.config['geodb_openeo']['postgrest_url']
         server_port = self.config['geodb_openeo']['postgrest_port']
-        client_id = self.config['geodb_openeo']['client_id']
-        client_secret = self.config['geodb_openeo']['client_secret']
+        client_id = self.config['geodb_openeo']['client_id'] \
+            if 'client_id' in self.config['geodb_openeo'] \
+            else os.getenv('XC_GEODB_OPENEO_CLIENT_ID')
+        client_secret = self.config['geodb_openeo']['client_secret'] \
+            if 'client_secret' in self.config['geodb_openeo'] \
+            else os.getenv('XC_GEODB_OPENEO_CLIENT_SECRET')
```

Comment:
Refactor out `api_config = self.config['geodb_openeo']` to improve readability and reduce code duplication.

Reply:
Of course. Done.